child_compassion/__openerp__.py

```diff
@@ -56,7 +56,11 @@
         'data/delegate_child_compassion_cron.xml',
         'data/child_compassion_template.xml',
     ],
-    'demo': [],
+    'demo': [
+        'demo/compassion.country.csv',

```

Comment:
The problem is for the id in the xml.
I need to update them in sbc_compassion (for the languages)